src/v2/components/Blocks/index.jsx

```diff
@@ -16,27 +14,12 @@ import useStyles from './styles';
 
 const BlocksPage = ({match}: {match: Match}) => {
   const classes = useStyles();
-  const {blocks, blockCount, start, next, prev} = BlocksTimelineStore;
+  const {blocks, blockCount, start} = BlocksTimelineStore;
 
   if (start !== match.params.start) {
     BlocksTimelineStore.init({start: match.params.start});
   }
 
-  const nav = (
```

Comment:
can we agree, please don't remove nav until we have a replacement? It's useful for debugging pagination